pages/participantinfo.tsx

```diff
@@ -404,48 +996,106 @@ export default function ParticipantInfo({
     const getProgressBarValue = (pageNum) =>
         progressBarIncrement * (pageNum + 1);
 
-    const getFormButton = () => {
-        if (pageNum === totalPages) {
-            return;
-        } else if (pageNum === totalPages - 1) {
-            return (
-                <FormButton
-                    onClick={() => setPageNum((prevPage) => prevPage + 1)}
-                >
-                    Finish
-                </FormButton>
-            );
-        } else if (pageNum === proofOfIncomePage) {
-            return (
-                <Box>
-                    <HStack spacing="24px">
-                        <FormButton>Upload Proof of Income</FormButton>
-                        <Button
-                            variant="ghost"
-                            as="u"
-                            onClick={() =>
-                                setPageNum((prevPage) => prevPage + 1)
-                            }
-                            borderRadius={100}
-                        >
-                            Skip for Now
-                        </Button>
-                    </HStack>
-                </Box>
-            );
-        }
-        return (
-            <FormButton
-                onClick={() => {
-                    setPageNum((prevPage) => prevPage + 1);
-                    window.scrollTo({ top: 0 });
-                }}
-            >
-                Next
-            </FormButton>
-        );
+    async function updateUser() {
+        const parentData: ParentInput = {
+            phoneNumber: parentPhoneNumber,
+            isLowIncome: undefined, // TODO
+            preferredLanguage: locale.en,
+            proofOfIncomeLink: undefined, // TODO
+            heardFrom: heardFromOptions.sort(),
+
+            childFirstName: participantFirstName,
+            childLastName: participantLastName,
+            childDateOfBirth: new Date(dateOfBirth),
+            addressLine1: address1,
+            addressLine2: address2,
+            postalCode: postalCode,
+            cityName: city,
+            province: participantProvince,
+            school: school,
+            grade: parseInt(grade, 10),
+
+            difficulties: participantDifficulties.sort(),
```

Comment:
Why do we need to sort it?

Reply:
We don't, I just figured it would be easier to read in the database and since the list is very small it would come at a marginal cost. I could remove it though if you prefer?

Reply:
I think that's fine to keep